modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -203,6 +205,13 @@ private PathItem _createPathItem(
 		if (operation != null) {
 			return new PathItem() {
 				{
+					if (operations.get(HttpMethod.DELETE) != null) {
```

Comment:
Maybe we should move this to the level of the previous if clause to keep the same format as with the GET operation

Reply:
Even extracting the common code to a new private function and then, invoke it 3 times with the different http methods

Reply:
If I understood well you mean that this should be move inside the GET operation?. It is that, it couldn't be possible because in the GET operation it only uses the `currentObjectEntryId` and the `relationName` parameters. However, the PUT method uses the same parameters (`currentObjectEntryId`, `relationName` and `relatedObjectEntryId`) as in the DELETE method